olp-cpp-sdk-core/include/olp/core/geo/coordinates/GeoCoordinates3d.h

```diff
@@ -178,9 +178,9 @@ class CORE_API GeoCoordinates3d {
   bool IsValid() const;
 
  protected:
-  GeoCoordinates geo_coordinates_;
-  double altitude_;
-  static const double kNaN_;
+  GeoCoordinates geo_coordinates_;/*!< 2D geodetic coordinates */
```

Comment:
```suggestion
  /// 2D geodetic coordinates
  GeoCoordinates geo_coordinates_;
```

This is what we used elsewhere.

Reply:
done